src/data/maps.js

```diff
@@ -20,11 +20,30 @@ const {
   WHEAT,
 } = cropType
 
-export const recipesMap = Object.keys(recipes).reduce((acc, recipeName) => {
-  const recipe = recipes[recipeName]
-  acc[recipe.id] = recipe
-  return acc
-}, {})
+export const kitchenRecipesMap = []
+export const forgeRecipesMap = []
+
+for (const recipeId of Object.keys(recipes)) {
+  const recipe = recipes[recipeId]
+
+  switch (recipe.recipeType) {
+    case recipeType.KITCHEN:
+      kitchenRecipesMap[recipe.id] = recipe
```

Comment:
did you choose to be explicit with separate lists on purpose? it would simplify this to make a data object that is keyed by `recipe.recipeType` instead, something like:

```
recipes = {
    [recipeType.KITCHEN]: [],
    [recipeType.FORGE]: [],
}```

the loop could actually build this out without requiring any defaults, and you can still ultimately have the same `recipesMap` export. this would just simplify the maintenance on this loop, but maybe that was intentional which is why i asked that first